stylesheets/_component.lists.scss

```diff
@@ -13,6 +13,10 @@
         color: color(primary, alt);
         text-decoration: none;
     }
+
+    & > a:focus {
+        color: color(text);
```

Comment:
Is this needed in addition to standard focus styles?

Reply:
May not be required now based on #1186